app-backend/datamodel/src/main/scala/com/azavea/rf/datamodel/Project.scala

```diff
@@ -19,6 +21,9 @@ case class Project(
   description: String,
   visibility: Visibility,
   tileVisibility: Visibility,
+  isAOIProject: Boolean,
+  aoiCadence: Long, /* Milliseconds */
```

Comment:
I think that still makes sense to keep `isAoiProject` because the frontend will be able to quickly check that.

Putting the cadence and last check time on the join table might be better in this case. Yeah, I remember I was a little worried about the checking logic for the process of checking if there are scenes to add. However, thinking about the user-facing API now putting it on project puts a burden on users that they shouldn't have unless we right a decoder.

I'm in favor of moving it to the join table, what about you @fosskers ?